alf/environments/suite_carla.py

```diff
@@ -745,6 +792,14 @@ def get_current_time_step(self, current_frame):
                 Player.PENALTY_RATE_RED_LIGHT * max(0., self._episode_reward))
         self._prev_violated_red_light_id = red_light_id
 
+        overspeed = self.get_overspeed_amount()
+        if overspeed > 0:
+            logging.info("actor=%d frame=%d OVERSPEED" % (self._actor.id,
+                                                          current_frame))
+            reward_vector[Player.REWARD_OVERSPEED] = 1.
+            info['overspeed'] = np.float32(1.0)
+            reward -= self._overspeed_penalty_weight * overspeed
```

Comment:
To make this invariant to delta_seconds, it needs to be multiplied with ``self._delta_seconds`` thus with same unit as distance.

Reply:
Good point. Done.